### PR TITLE
Functionalities to work on console connection using PyEZ 2.0

### PR DESCRIPTION
**Pull request by vnitinv**
example:
To take mode and port. 

```yaml
  - name: Checking NETCONF connectivity
    wait_for: host={{ inventory_hostname }} timeout=5
  - name: Get configs
    junos_get_config:
      user: "{{ USERNAME }}"
      passwd: "{{ PASSWORD }}"
      port: 7011
      mode: "telnet"
      logfile: get_config.log
      format: xml
```

### CODE REVIEW

File: library/junos_cli [hunk 1]
```diff
@@ -63,9 +63,14 @@ options:
...
         description:
-            - TCP port number to use when connecting to the device
+            - port number to use when connecting to the device
         required: false
         default: 830
+    mode:
+        description:
+            - mode of console connection (telnet/serial)
+        required: false
+        default: None
```
Comment (ydnath) [inline library/junos_cli]: You may want to indicate that default is regular SSH connection

File: library/junos_cli [hunk 2]
```diff
@@ -101,6 +106,15 @@ EXAMPLES = '''
    logfile: cli.log
    dest: "{{ inventory_hostname }}.xml"
    format: xml
+
+# Run cli over console connection via PyEZ
```
Comment (ydnath) [inline library/junos_cli]: Mode is telnet but comment says console (indicating serial). Please calrify.

File: library/junos_cli (hunk 1 above)
Comment (vnitinv) [inline library/junos_cli]: Done with changes. Please review.

File: library/junos_cli (hunk 2 above)
Comment (vnitinv) [inline library/junos_cli]: Done with changes. Please review.

**Pull request merged**